pkg/apis/apps/types.go

```diff
@@ -157,6 +184,13 @@ type StatefulSetSpec struct {
 	// consists of all revisions not represented by a currently applied
 	// StatefulSetSpec version. The default value is 10.
 	RevisionHistoryLimit *int32
+
+	// PersistentVolumeClaimDeletePolicy indicates the policy that will
+	// determine when volumes from the VolumeClaimTemplates will be deleted when
+	// the controlling StatefulSet is deleted or scaled down. This requires
+	// the StatefulSetAutoDeletePVC feature gate.
```

Comment:
You should say that this feature is alpha level.

Reply:
This had been done back in March.